<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MITTransposeBase</class>
 <widget class="QWidget" name="MITTransposeBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MusE: Midi Input Plugin: Transpose</string>
  </property>
  <layout class="QHBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>11</number>
   </property>
   <item>
    <widget class="QCheckBox" name="onCheckBox">
     <property name="text">
      <string>On</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="TextLabel1">
     <property name="text">
      <string>TriggerKey</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="indent">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="MusEGui::PitchEdit" name="triggerKeySpinBox" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="TextLabel2">
     <property name="text">
      <string>Transpose:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="indent">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="transposeLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="midLineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string>+0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
     <property name="indent">
      <number>5</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MusEGui::PitchEdit</class>
   <extends>QWidget</extends>
   <header>pitchedit.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">pitchedit.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
